src/index.js

```diff
@@ -7,43 +7,61 @@ import transformCall from './transformers/call';
 import transformImport from './transformers/import';
 
 
-const defaultBabelExtensions = ['.js', '.jsx', '.es', '.es6'];
-export const defaultExtensions = defaultBabelExtensions;
```

Comment:
I moved handling extensions into the `normalizePluginOptions` function, so maybe they are not needed anymore? After calling that function you can just take the extensions from the options, so the "default extensions" logic can be romved from the plugin. Win-win :)